packages/react-vapor/setupTests/assetsTransformer.js

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+
+module.exports = {
+    process(src, filename, config, options) {
+        return 'module.exports = ' + JSON.stringify(path.basename(filename)) + ';';
```

Comment:
Would it be easier to read `return `module.exports = ${JSON.stringify(path.basename(filename))};`;` ?